packages/application-components/src/components/dialogs/confirmation-dialog/confirmation-dialog.visualroute.js

```diff
@@ -14,9 +14,9 @@ const ConfirmationDialogExample = props => (
       title="Lorem ipsum"
       size={props.size}
       isOpen={true}
-      onClose={() => {}}
-      onCancel={() => {}}
-      onConfirm={() => {}}
+      onClose={() => undefined}
```

Comment:
I like this eslint rule requiring to make it explicit that a function returns undefined.